main.go

```diff
@@ -272,6 +281,40 @@ func checkArgs(_ *corev2.Event) error {
 	return nil
 }
 
+func mergeToEmail(event *corev2.Event) []string {
+	toEmailKey := config.PluginConfig.Keyspace + "/toEmail/merge"
```

Comment:
for the annotation, consider using sensu.io/plugins/email/config/toEmail/addresses instead